src/hyperion/parameters/gridscan.py

```diff
@@ -225,6 +152,10 @@ def FGS_params(self) -> GridScanParams:
 
     @property
     def panda_FGS_params(self) -> PandAGridScanParams:
+        if self.y_steps % 2 and self.z_steps > 0:
+            raise OddYStepsException(
+                "The number of Y steps must be even for a PandA gridscan"
+            )
```

Comment:
Must: I can comment this out and all tests still pass, can we make sure it's caught?